src/handlers/completion.rs

```diff
@@ -254,6 +260,24 @@ async fn find_dot_completions(
     })
 }
 
+pub fn get_specific_object(
+    name: String,
+    pos: Position,
+    uri: String,
+) -> Result<Vec<Arc<dyn Completable + Send + Sync>>, String> {
+    let pkg = utils::create_completion_package_removed(uri, pos)?;
+    let walker = Rc::new(walk::Node::Package(&pkg));
+    let mut visitor = CompletableObjectFinderVisitor::new(name);
+
+    walk::walk(&mut visitor, walker);
+
+    if let Ok(state) = visitor.state.lock() {
+        return Ok(state.completables.clone());
```

Comment:
borrow here

Reply:
since this is a return I would prefer to clone here